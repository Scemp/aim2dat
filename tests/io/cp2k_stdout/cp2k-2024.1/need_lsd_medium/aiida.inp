!!! Generated by AiiDA !!!
&FORCE_EVAL
   &DFT
      BASIS_SET_FILE_NAME BASIS_MOLOPT_UCL
      &KPOINTS
         FULL_GRID .FALSE.
         PARALLEL_GROUP_SIZE 0
         SCHEME MONKHORST-PACK 3 3 5
      &END KPOINTS
      &MGRID
         CUTOFF 600.0
         NGRIDS 5
         REL_CUTOFF 100.0
      &END MGRID
      &POISSON
         PERIODIC XYZ
         POISSON_SOLVER PERIODIC
      &END POISSON
      POTENTIAL_FILE_NAME GTH_POTENTIALS
      &PRINT
         &MO
            &EACH
               QS_SCF 1000
            &END EACH
            EIGENVALUES .TRUE.
         &END MO
      &END PRINT
      &QS
         EPS_DEFAULT 1e-12
         EXTRAPOLATION USE_GUESS
      &END QS
      ROKS .FALSE.
      &SCF
         ADDED_MOS 75
         EPS_SCF 5e-07
         MAX_SCF 250
         &MIXING
            ALPHA 0.01
            METHOD BROYDEN_MIXING
            NBUFFER 30
         &END MIXING
         SCF_GUESS RESTART
      &END SCF
      UKS .FALSE.
      &XC
         &XC_FUNCTIONAL PBE
         &END XC_FUNCTIONAL
      &END XC
   &END DFT
   METHOD Quickstep
   STRESS_TENSOR ANALYTICAL
   &SUBSYS
      &CELL
         A 16.273218457353 0.0             0.0            
         B -5.2244224653624 16.83395132236  0.0            
         C -4.3207234542062 4.6146955935651 12.175262387007
         PERIODIC XYZ
      &END CELL
      &KIND Cs
         BASIS_SET TZVP-MOLOPT-SR-GTH
         POTENTIAL GTH-PBE-q9
      &END KIND
      &TOPOLOGY
         COORD_FILE_FORMAT XYZ
         COORD_FILE_NAME aiida.coords.xyz
      &END TOPOLOGY
   &END SUBSYS
&END FORCE_EVAL
&GLOBAL
   PROJECT aiida
   RUN_TYPE energy
&END GLOBAL

