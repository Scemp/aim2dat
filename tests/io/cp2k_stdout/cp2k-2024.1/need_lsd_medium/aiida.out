 DBCSR| CPU Multiplication driver                                           XSMM (U)
 DBCSR| Multrec recursion limit                                              512 (U)
 DBCSR| Multiplication stack size                                           1000 (D)
 DBCSR| Maximum elements for images                                    UNLIMITED (U)
 DBCSR| Multiplicative factor virtual images                                   1 (U)
 DBCSR| Use multiplication densification                                       T (D)
 DBCSR| Multiplication size stacks                                             3 (U)
 DBCSR| Use memory pool for CPU allocation                                     F (U)
 DBCSR| Number of 3D layers                                               SINGLE (U)
 DBCSR| Use MPI memory allocation                                              F (U)
 DBCSR| Use RMA algorithm                                                      F (U)
 DBCSR| Use Communication thread                                               T (U)
 DBCSR| Communication thread load                                             87 (D)
 DBCSR| MPI: My process id                                                     0
 DBCSR| MPI: Number of processes                                              96
 DBCSR| OMP: Current number of threads                                         1
 DBCSR| OMP: Max number of threads                                             1
 DBCSR| Split modifier for TAS multiplication algorithm                  1.0E+00 (U)


  **** **** ******  **  PROGRAM STARTED AT               2024-07-08 08:07:02.802
 ***** ** ***  *** **   PROGRAM STARTED ON                                 c0266
 **    ****   ******    PROGRAM STARTED BY                              nichdsas
 ***** **    ** ** **   PROGRAM PROCESS ID                                869426
  **** **  *******  **  PROGRAM STARTED IN /scratch-grete/usr/nichdsas/test_pars
                                           er/cp2k-2024.1/need_lsd_medium

 CP2K| version string:                                       CP2K version 2024.1
 CP2K| source code revision number:                                             
 CP2K| cp2kflags: omp libint fftw3 libxc elpa parallel scalapack xsmm spglib sir
 CP2K|            ius libvori libbqb
 CP2K| is freely available from                            https://www.cp2k.org/
 CP2K| Program compiled at                                   2024-05-28 17:35:28
 CP2K| Program compiled on                                                      
 CP2K| Program compiled for                                               x86_64
 CP2K| Data directory path    /sw/rev/24.05/sapphirerapids_opa_rocky8/linux-rock
 CP2K| Input file name                                                 aiida.inp

 GLOBAL| Force Environment number                                              1
 GLOBAL| Basis set file name                                    BASIS_MOLOPT_UCL
 GLOBAL| Potential file name                                      GTH_POTENTIALS
 GLOBAL| MM Potential file name                                     MM_POTENTIAL
 GLOBAL| Coordinate file name                                   aiida.coords.xyz
 GLOBAL| Method name                                                        CP2K
 GLOBAL| Project name                                                      aiida
 GLOBAL| Run type                                                         ENERGY
 GLOBAL| FFT library                                                       FFTW3
 GLOBAL| Diagonalization library                                            ELPA
 GLOBAL| DGEMM library                                                      BLAS
 GLOBAL| Minimum number of eigenvectors for ELPA usage                        16
 GLOBAL| Orthonormality check for eigenvectors                          DISABLED
 GLOBAL| Matrix multiplication library                                 ScaLAPACK
 GLOBAL| All-to-all communication in single precision                          F
 GLOBAL| FFTs using library dependent lengths                                  F
 GLOBAL| Grid backend                                                       AUTO
 GLOBAL| Global print level                                               MEDIUM
 GLOBAL| MPI I/O enabled                                                       T
 GLOBAL| Total number of message passing processes                            96
 GLOBAL| Number of threads for this process                                    1
 GLOBAL| This output is from process                                           0
 GLOBAL| Stack size for threads created by OpenMP (OMP_STACKSIZE)        default
 GLOBAL| CPU model name                           Intel(R) Xeon(R) Platinum 8468
 GLOBAL| CPUID                                                              1003

 MEMORY| system memory details [Kb]
 MEMORY|                        rank 0           min           max       average
 MEMORY| MemTotal            527678176     527678176     527678176     527678176
 MEMORY| MemFree             510848536     510848536     510848536     510848536
 MEMORY| Buffers                 11280         11280         11280         11280
 MEMORY| Cached                4108340       4108340       4108340       4108340
 MEMORY| Slab                  1395000       1395000       1395000       1395000
 MEMORY| SReclaimable           215996        215996        215996        215996
 MEMORY| MemLikelyFree       515184152     515184152     515184152     515184152


 *** Fundamental physical constants (SI units) ***

 *** Literature: B. J. Mohr and B. N. Taylor,
 ***             CODATA recommended values of the fundamental physical
 ***             constants: 2006, Web Version 5.1
 ***             http://physics.nist.gov/constants

 Speed of light in vacuum [m/s]                             2.99792458000000E+08
 Magnetic constant or permeability of vacuum [N/A**2]       1.25663706143592E-06
 Electric constant or permittivity of vacuum [F/m]          8.85418781762039E-12
 Planck constant (h) [J*s]                                  6.62606896000000E-34
 Planck constant (h-bar) [J*s]                              1.05457162825177E-34
 Elementary charge [C]                                      1.60217648700000E-19
 Electron mass [kg]                                         9.10938215000000E-31
 Electron g factor [ ]                                     -2.00231930436220E+00
 Proton mass [kg]                                           1.67262163700000E-27
 Fine-structure constant                                    7.29735253760000E-03
 Rydberg constant [1/m]                                     1.09737315685270E+07
 Avogadro constant [1/mol]                                  6.02214179000000E+23
 Boltzmann constant [J/K]                                   1.38065040000000E-23
 Atomic mass unit [kg]                                      1.66053878200000E-27
 Bohr radius [m]                                            5.29177208590000E-11

 *** Conversion factors ***

 [u] -> [a.u.]                                              1.82288848426455E+03
 [Angstrom] -> [Bohr] = [a.u.]                              1.88972613288564E+00
 [a.u.] = [Bohr] -> [Angstrom]                              5.29177208590000E-01
 [a.u.] -> [s]                                              2.41888432650478E-17
 [a.u.] -> [fs]                                             2.41888432650478E-02
 [a.u.] -> [J]                                              4.35974393937059E-18
 [a.u.] -> [N]                                              8.23872205491840E-08
 [a.u.] -> [K]                                              3.15774647902944E+05
 [a.u.] -> [kJ/mol]                                         2.62549961709828E+03
 [a.u.] -> [kcal/mol]                                       6.27509468713739E+02
 [a.u.] -> [Pa]                                             2.94210107994716E+13
 [a.u.] -> [bar]                                            2.94210107994716E+08
 [a.u.] -> [atm]                                            2.90362800883016E+08
 [a.u.] -> [eV]                                             2.72113838565563E+01
 [a.u.] -> [Hz]                                             6.57968392072181E+15
 [a.u.] -> [1/cm] (wave numbers)                            2.19474631370540E+05
 [a.u./Bohr**2] -> [1/cm]                                   5.14048714338585E+03
 

 CELL_TOP| Volume [angstrom^3]:                                      3335.322637
 CELL_TOP| Vector a [angstrom    16.273     0.000     0.000   |a| =    16.273218
 CELL_TOP| Vector b [angstrom    -5.224    16.834     0.000   |b| =    17.626018
 CELL_TOP| Vector c [angstrom    -4.321     4.615    12.175   |c| =    13.718640
 CELL_TOP| Angle (b,c), alpha [degree]:                                65.504700
 CELL_TOP| Angle (a,c), beta  [degree]:                               108.357962
 CELL_TOP| Angle (a,b), gamma [degree]:                               107.241747
 CELL_TOP| Numerically orthorhombic:                                          NO
 CELL_TOP| Periodicity                                                       XYZ

 GENERATE|  Preliminary Number of Bonds generated:                             0
 GENERATE|  Achieved consistency in connectivity generation.

 CELL| Volume [angstrom^3]:                                          3335.322637
 CELL| Vector a [angstrom]:      16.273     0.000     0.000   |a| =    16.273218
 CELL| Vector b [angstrom]:      -5.224    16.834     0.000   |b| =    17.626018
 CELL| Vector c [angstrom]:      -4.321     4.615    12.175   |c| =    13.718640
 CELL| Angle (b,c), alpha [degree]:                                    65.504700
 CELL| Angle (a,c), beta  [degree]:                                   108.357962
 CELL| Angle (a,b), gamma [degree]:                                   107.241747
 CELL| Numerically orthorhombic:                                              NO
 CELL| Periodicity                                                           XYZ

 CELL_REF| Volume [angstrom^3]:                                      3335.322637
 CELL_REF| Vector a [angstrom    16.273     0.000     0.000   |a| =    16.273218
 CELL_REF| Vector b [angstrom    -5.224    16.834     0.000   |b| =    17.626018
 CELL_REF| Vector c [angstrom    -4.321     4.615    12.175   |c| =    13.718640
 CELL_REF| Angle (b,c), alpha [degree]:                                65.504700
 CELL_REF| Angle (a,c), beta  [degree]:                               108.357962
 CELL_REF| Angle (a,b), gamma [degree]:                               107.241747
 CELL_REF| Numerically orthorhombic:                                          NO
 CELL_REF| Periodicity                                                       XYZ

 *******************************************************************************
                                    Kpoints
 *******************************************************************************
 BRILLOUIN| K-point scheme                                        Monkhorst-Pack
 BRILLOUIN| K-Point grid                                             3    3    5
 BRILLOUIN| Accuracy in Symmetry determination                      0.100000E-05
 BRILLOUIN| K-Point point group symmetrization                               OFF
 BRILLOUIN| Wavefunction type                                            COMPLEX
 BRILLOUIN| List of Kpoints [2 Pi/Bohr]                                       23
 BRILLOUIN| Number           Weight            X              Y              Z
 BRILLOUIN|     1           0.04444       -0.33333       -0.33333       -0.40000
 BRILLOUIN|     2           0.04444       -0.33333       -0.33333       -0.20000
 BRILLOUIN|     3           0.04444       -0.33333       -0.33333        0.00000
 BRILLOUIN|     4           0.04444       -0.33333       -0.33333        0.20000
 BRILLOUIN|     5           0.04444       -0.33333       -0.33333        0.40000
 BRILLOUIN|     6           0.04444       -0.33333        0.00000       -0.40000
 BRILLOUIN|     7           0.04444       -0.33333        0.00000       -0.20000
 BRILLOUIN|     8           0.04444       -0.33333        0.00000        0.00000
 BRILLOUIN|     9           0.04444       -0.33333        0.00000        0.20000
 BRILLOUIN|    10           0.04444       -0.33333        0.00000        0.40000
 BRILLOUIN|    11           0.04444       -0.33333        0.33333       -0.40000
 BRILLOUIN|    12           0.04444       -0.33333        0.33333       -0.20000
 BRILLOUIN|    13           0.04444       -0.33333        0.33333        0.00000
 BRILLOUIN|    14           0.04444       -0.33333        0.33333        0.20000
 BRILLOUIN|    15           0.04444       -0.33333        0.33333        0.40000
 BRILLOUIN|    16           0.04444        0.00000       -0.33333       -0.40000
 BRILLOUIN|    17           0.04444        0.00000       -0.33333       -0.20000
 BRILLOUIN|    18           0.04444        0.00000       -0.33333        0.00000
 BRILLOUIN|    19           0.04444        0.00000       -0.33333        0.20000
 BRILLOUIN|    20           0.04444        0.00000       -0.33333        0.40000
 BRILLOUIN|    21           0.04444        0.00000        0.00000       -0.40000
 BRILLOUIN|    22           0.04444        0.00000        0.00000       -0.20000
 BRILLOUIN|    23           0.02222        0.00000        0.00000        0.00000
 *******************************************************************************

 *******************************************************************************
 *******************************************************************************
 **                                                                           **
 **     #####                         ##              ##                      **
 **    ##   ##            ##          ##              ##                      **
 **   ##     ##                       ##            ######                    **
 **   ##     ##  ##   ##  ##   #####  ##  ##   ####   ##    #####    #####    **
 **   ##     ##  ##   ##  ##  ##      ## ##   ##      ##   ##   ##  ##   ##   **
 **   ##  ## ##  ##   ##  ##  ##      ####     ###    ##   ######   ######    **
 **    ##  ###   ##   ##  ##  ##      ## ##      ##   ##   ##       ##        **
 **     #######   #####   ##   #####  ##  ##  ####    ##    #####   ##        **
 **           ##                                                    ##        **
 **                                                                           **
 **                                                ... make the atoms dance   **
 **                                                                           **
 **            Copyright (C) by CP2K developers group (2000-2024)             **
 **                      J. Chem. Phys. 152, 194103 (2020)                    **
 **                                                                           **
 *******************************************************************************

 *******************************************************************************
 *   ___                                                                       *
 *  /   \                                                                      *
 * [ABORT]                                                                     *
 *  \___/            Use the LSD option for an odd number of electrons         *
 *    |                                                                        *
 *  O/|                                                                        *
 * /| |                                                                        *
 * / \                                                   qs_environment.F:1407 *
 *******************************************************************************


 ===== Routine Calling Stack ===== 

            2 qs_init_subsys
            1 CP2K
